django/template/base.py

```diff
@@ -254,15 +255,20 @@ def tokenize(self):
         """
         Return a list of tokens from a given template_string.
         """
-        in_tag = False
-        result = []
-        for bit in tag_re.split(self.template_string):
-            if bit:
-                result.append(self.create_token(bit, in_tag))
-            in_tag = not in_tag
+        result, upto = [], 0
+        for match in tag_re.finditer(self.template_string):
+            start, end = match.span()
+            if start > upto:
+                result.append(self.create_token(self.template_string[upto:start], (upto, start), False))
```

Comment:
Passing the last argument as a keyword argument (`in_tag=False`) would make the code easier to follow.

This is almost always better for boolean arguments, except when they're the only argument of the function.